test/ERC6538Registry.t.sol

```diff
@@ -55,6 +56,32 @@ contract ERC6538RegistryTest is Test, Deploy {
     vm.assume(_address != address(vm));
     vm.assume(_address != address(address(0x000000000000000000636F6e736F6c652e6c6f67)));
   }
+
+  function manipulateSignature(bytes memory signature) public pure returns (bytes memory) {
+    (uint8 v, bytes32 r, bytes32 s) = splitSignature(signature);
+
+    uint8 manipulatedV = v % 2 == 0 ? v - 1 : v + 1;
```

Comment:
Some comments might be helpful in these functions, e.g. why do we have to change v to change s?

Reply:
My bad, these functions are from https://medium.com/draftkings-engineering/signature-malleability-7a804429b14a